pkgs/build-support/node/fetch-npm-deps/default.nix

```
@@ -0,0 +1,52 @@
+{ lib, stdenv, rustPlatform, Security }:
```

Just for consistency, apologies about spamming my review with these :sweat_smile: 
```suggestion
{ lib
, stdenv
, rustPlatform
, Security
}:
```